doc/docs.md

```diff
@@ -16,14 +16,58 @@ you can do in V.
 
 ## Table of Contents
 
-<table>
-	<tr>
-		<td><a href='#hello-world'>1. Hello world</a></td>
-		<td><a href='#comments'>2. Comments</a></td>
-		<td><a href='#functions'>3. Functions</a></td>
-		<td><a href='#'>TODO</a></td>
-	</tr>
-</table>
+<ol>
```

Comment:
I created a table, so that it wouldn't take so much vertical space

Reply:
i can edit it but ol have automatic numbers on left side

Reply:
but i will change it

Reply:
i fixed it but with a table it isn't look nice